junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/discovery/DiscoverySelectorResolver.java

```diff
@@ -24,11 +24,8 @@
 import org.junit.platform.commons.meta.API;
 import org.junit.platform.engine.EngineDiscoveryRequest;
 import org.junit.platform.engine.TestDescriptor;
-import org.junit.platform.engine.discovery.ClasspathRootSelector;
-import org.junit.platform.engine.discovery.JavaClassSelector;
-import org.junit.platform.engine.discovery.JavaMethodSelector;
-import org.junit.platform.engine.discovery.JavaPackageSelector;
-import org.junit.platform.engine.discovery.UniqueIdSelector;
+import org.junit.platform.engine.discovery.*;
```

Comment:
I think this would be clearer as a bunch of explicit imports, rather than a wildcard.